### ISSUE TITLE
Squeezer wont form 0.10.57 ver 1.10.2

### ISSUE THREAD

**Issue by Dragonracer73** [BluSunrize/ImmersiveEngineering#2041]
Hello Playing a modpack created by the League of Ordinary Gamers called Coop Life (pack was brought to life by Vaygrim) I tend to lay ideas for base builds out in creative (though I do know your machines need survival to form) while trying to make the squeezer I notice the picture does not include the piston on the final step in the book. as well as losing the text on the page. I also get a stack overflow error msg. I am including screenshots of the bug. 
The modpack file(compressed) is to big to send here, but can be found on the twitch launcher. Coop Life
![2017-04-27_07 46 16](https://cloud.githubusercontent.com/assets/15730038/25484391/6ead80c6-2b1f-11e7-8dda-19bb80a76ba1.png)
![2017-04-27_07 38 11](https://cloud.githubusercontent.com/assets/15730038/25484395/7327f582-2b1f-11e7-9bb0-fa62d01a7192.png)
 Your help in this matter would be greatly appreciated. 
sincerily, Dragonracer73

**Comment by malte0811** [comment]
This is probably an actual issue, but the issue is not the multiblock not forming. Something is breaking the manual rendering very badly, potentially some mod overriding the piston render/model. There are probably "some" errors in your log (more like one every frame, you can find the log file in `<your MC folder>/logs/fml-client-latest.log`). It would be nice if you could upload that file as well (or, if the log is very long, the last ~200 lines).
BTW, this is the full multiblock: (you have added the piston, but I don't think you added the second steel fence)
![2017-04-27_16 24 01](https://cloud.githubusercontent.com/assets/10406104/25487956/ffb45b08-2b65-11e7-805a-971af5204c84.png)

**Comment by Dragonracer73** [comment]
[I
[Immersive issue.zip](https://github.com/BluSunrize/ImmersiveEngineering/files/962938/Immersive.issue.zip)
here is a list of the errors I am getting

**Comment by BluSunrize** [comment]
It was specifically asked for the client log.
We need to see that for modlist and system specs. Upload the contents of the client-latest file as described above to pastebin.com or a similar website. Zipfiles with partial text documents don't help in the slightest.

**Comment by Dragonracer73** [comment]
I have forced a crash, and saved the report, as well as copied it to gist, but unsure of how to get it here. a zip file of the crash report is no good?

**Comment by theunkn0wn1** [comment]
You just paste the link to the gist here, the same url you use to access it yourself (iirc)

**Comment by BluSunrize** [comment]
I said Pastebin. Gist also works. Either produce a link.

The problem with zip files is the potentially malicious content, which is a risk I'd like to not have on my issuetracker, kthx.

**Comment by malte0811** [comment]
The zip files are ok (for me at least, even though gist would have been better). You should try removing Optifine, it is not supported as stated in the contributing guidelines. If that does not fix it, it is probably caused by CodeChickenLib, but I will have to do some debugging to find out the details.

**Comment by BluSunrize** [comment]
Dang Malte, we were close there =P
Yeah I was gonna call out Optifine as well, remove that, see if the issue persists.

**Comment by Dragonracer73** [comment]
actually, I have found, (through the league) that removing the texture pack (I was using sphax) seems to have fixed the issue. I removed the texture pack, and everything returned to normal. I apologize for wasting your time. I do appreciate the help, and really enjoy this mod, since it inception in 1.7. Also know I would never send malicious content, but I understand the reasons. 
Again I thank you all for the help in this matter.
